lib/spack/docs/basic_usage.rst

```diff
@@ -1222,11 +1026,11 @@ Module Commands for Shell Scripts
 
 Although Spack is flexible, the ``module`` command is much faster.
 This could become an issue when emitting a series of ``spack load``
-commands inside a shell script.  By adding the ``--shell`` flag,
-``spack module find`` may also be used to generate code that can be
-cut-and-pasted into a shell script.  For example:
+commands inside a shell script.  The ``spack module loads`` may also
+be used to generate code that can be cut-and-pasted into a shell
+script.  For example:
 
-.. code-block:: console
+.. code-block:: sh
```

Comment:
@adamjstewart recently discovered this in one of the issues related to his big documentation PR.  it think it's worth adding to the developer guide.  I didn't know about it but I think it's quite useful.

I really wish there was a good way to get ANSI colors in the docs.